fdbclient/include/fdbclient/Tenant.h

```diff
@@ -61,6 +61,7 @@ struct TenantMapEntry {
 	Key prefix;
 	TenantName tenantName;
 	TenantAPI::TenantLockState tenantLockState = TenantAPI::TenantLockState::UNLOCKED;
+	Optional<UID> tenantLockId;
```

Comment:
Can you add an equality check for this in `operator==`?

Reply:
done